mirascope/core/base/types.py

```diff
@@ -0,0 +1,22 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pydub import AudioSegment
+
+    has_pydub_module: bool
+else:
+    try:
+        from pydub import AudioSegment  # pyright: ignore [reportAssignmentType]
```

Comment:
How do you feel this file name?

Reply:
i think it's fine, we should also move the `Image.Image` thing here too then right?